test/utils-safe.js

```diff
@@ -184,6 +184,41 @@ const deploySafe = async (
     ).then((e) => e.wait());
 };
 
+/**
+ * Sign a safe tx with one signer using EIP 191 signature
+ */
+const signSafeTx = async (safeInstance, safeTxParams, signer) => {
```

Comment:
https://github.com/safe-global/safe-smart-account/blob/914d0f8fab0e8f73ef79581f6fbce86e34b049c3/src/utils/execution.ts#L112

Check if it's possible to simplify signing safe tx